hail/python/hailtop/aiocloud/aiogoogle/client/storage_client.py

```diff
@@ -290,10 +316,7 @@ async def insert_object(self, bucket: str, name: str, **kwargs) -> WritableStrea
         assert 'name' not in params
         params['name'] = name
 
-        if 'data' in params:
-            return await self._session.post(
-                f'https://storage.googleapis.com/upload/storage/v1/b/{bucket}/o',
-                **kwargs)
+        assert 'data' not in params
```

Comment:
I assume we never reach this case in our current code...

Reply:
Yeah, it's totally unused.